framework_tvb/tvb/adapters/analyzers/wavelet_adapter.py

```diff
@@ -62,16 +62,37 @@ class WaveletAdapterModel(ViewModel, ContinuousWaveletTransform):
         doc="""The timeseries to which the wavelet is to be applied."""
     )
 
+    f_lo = Attr(
+        field_type=float,
+        required=True,
+        label='Lo',
+        default=ContinuousWaveletTransform.frequencies.default.lo,
+        doc='start of range'
+    )
+
+    f_hi = Attr(
+        field_type=float,
+        required=True,
+        label='Hi',
+        default=ContinuousWaveletTransform.frequencies.default.hi,
+        doc='end of range'
+    )
+
+    f_step = Attr(
+        field_type=float,
+        required=True,
+        label='Step',
+        default=ContinuousWaveletTransform.frequencies.default.step,
+        doc='step of range'
+    )
+
```

Comment:
I think these new fields are not needed on the model class, we can define them directly in the Form where they are used. Otherwise, they will be written separately in the H5 file, while we already write the Range.

Reply:
Also, you can use Float instead of Attr so you don't need to define Attr(field_type=float).
required is True by default so that is not necessary either,